cdap-cli/src/main/java/co/cask/cdap/cli/command/DescribeStreamCommand.java

```diff
@@ -50,13 +50,14 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
     StreamProperties config = streamClient.getConfig(streamId);
 
     new AsciiTable<StreamProperties>(
-      new String[] { "name", "ttl", "format", "schema" },
+      new String[] { "streamId", "ttl", "format", "schema" },
       Lists.newArrayList(config),
       new RowMaker<StreamProperties>() {
         @Override
         public Object[] makeRow(StreamProperties object) {
           FormatSpecification format = object.getFormat();
-          return new Object[] { object.getName(), object.getTTL(), format.getName(), format.getSchema().toString() };
+          return new Object[] { object.getStreamId().toString(), object.getTTL(),
```

Comment:
Ok, I am removing namespace/name from StreamProperties anyways, so this will be gone.
Removal of name in this PR: https://github.com/caskdata/cdap/pull/1282